### ISSUE TITLE
Cannot activate the "Debugger for Unity" extension because it depends on an unknown 'ms-vscode.csharp' extension

### ISSUE THREAD

**Issue by jcarcamo** [Unity-Technologies/vscode-unity-debug#154]
VS code is not recognizing the extension anymore. Apparently ```ms-vscode.csharp``` was renamed to ```ms-dotnettools.csharp```

![image](https://user-images.githubusercontent.com/3542741/76013315-b70fa900-5ee5-11ea-8379-9f07f465159f.png)

OS: Windows 10 Pro Version 1903
VS Code version: 1.42.1

**Comment by ChrisWeiss** [comment]
This is addressed in PR #152 
Similar to #153

**Comment by zachstronaut** [comment]
We need a release to the marketplace ASAP.

**Comment by S1Zr** [comment]
In deed.

**Comment by MatheusSolar** [comment]
ok, i followed the other posts referenced above and they suggested to change the dependencies on the "json" file for the unity debugger, i did that but VSCode was requesting to install the extension for c#. 
so, i kind of solved the problem by changing the "package.json" and the manifest file for the c# extension of VSCode, basically change the publisher from the new  "ms-dotnettools.csharp" to the old "ms-vscode.csharp" and everything works normal now,

**Comment by Kotsuha** [comment]
The 'C#' extension changes its name from `ms-vscode.csharp` to `ms-dotnettools.csharp` in `1.21.13`. I naively thought maybe I can somehow tell 'Debugger for Unity' the new name of 'C#'. I searched in `VSCode\data\extensions\unity.unity-debug-2.7.2` folder and found that there are two files containing the word `ms-vscode.csharp`:
* VSCode\data\extensions\unity.unity-debug-2.7.2\.vsixmanifest
* VSCode\data\extensions\unity.unity-debug-2.7.2\package.json

So I changed them.

And yeah it does not show the warning anymore but I still cannot debug Unity :/

So I changed back to using `ms-vscode.csharp-1.21.11` and uncheck **Extensions: Auto Update** like the link said:

> Pending a marketplace release of this Unity VS extension update, you can roll back C# extension:
> 
> [OmniSharp/omnisharp-vscode#3620 (comment)](https://github.com/OmniSharp/omnisharp-vscode/issues/3620#issuecomment-595458417)

It works now.
